azure-functions/acbs-function/mappings/deal/helpers/get-deal-submission-date.js

```diff
@@ -0,0 +1,3 @@
+const getDealSubmissionDate = (deal) => deal.dealSnapshot.submissionDate || deal.dealSnapshot.details.submissionDate;
```

Comment:
One for the future/future improvement - it'd be nice to use `mapSubmittedDeal` for these functions as then wouldn't need these conditions

Reply:
Will do